doc_source/tools.md

```diff
@@ -104,15 +106,15 @@ If your app has a single stack, you don't have to specify the stack name\.
 
 If a `cdk.json` or `~/.cdk.json` file exists, options specified there are used as defaults\. Settings in `cdk.json` take precedence\.
 
-### Bootstrapping the CDK<a name="tools_bootstrap"></a>
+### Bootstrapping the AWS CDK<a name="tools_bootstrap"></a>
```

Comment:
This is not about bootstrapping the CDK, it's about bootstrapping an AWS environment (account/region) so that the CDK CLI will be able to deploy apps into it.

Reply:
I'll update the description.